pkgs/applications/misc/regextester/default.nix

```diff
@@ -3,47 +3,51 @@
 , gettext
 , libxml2
 , pkgconfig
-, gtk3
+, glib
 , granite
+, gtk3
 , gnome3
-, cmake
+, meson
 , ninja
+, gobjectIntrospection
+, gsettings-desktop-schemas
 , vala
-, elementary-cmake-modules
 , wrapGAppsHook }:
 
 stdenv.mkDerivation rec {
   name = "regextester-${version}";
-  version = "0.1.7";
+  version = "1.0.1";
 
   src = fetchFromGitHub {
     owner = "artemanufrij";
     repo = "regextester";
     rev = version;
-    sha256 = "07shdm10dc7jz2hka5dc51yp81a0dgc47nmkrp6fs6r9wqx0j30n";
+    sha256 = "1xwwv1hccni1mrbl58f7ly4qfq6738vn24bcbl2q346633cd7kx3";
   };
 
-  XDG_DATA_DIRS = stdenv.lib.concatStringsSep ":" [
-    "${granite}/share"
-    "${gnome3.libgee}/share"
-  ];
-
   nativeBuildInputs = [
     pkgconfig
-    wrapGAppsHook
-    vala
-    cmake
+    meson
     ninja
     gettext
+    gobjectIntrospection
     libxml2
-    elementary-cmake-modules
+    vala
+    wrapGAppsHook
   ];
   buildInputs = [
-    gtk3
+    glib
```

Comment:
I prefer to be explicit and include every library that the project links with – the project can stop using `gtk` or `gtk` can stop propagating `glib` any time. (Well in the case of gtk – glib relationship, it pretty unlikely but I do not like making exceptions.)

Reply:
I will keep that in mind in the future as well.

Reply:
To be clear, I meant listing in `buildInputs` all the libraries declared as `dependency` in `meson.build`, not the transitive dependencies. (Though projects sometimes forget to list some dependencies, relying on transitivity, which leads to breakages like https://github.com/NixOS/nixpkgs/issues/36468)

Reply:
So just gtk3 would be appropriate because [`meson.build`](https://github.com/artemanufrij/regextester/blob/128f16396230d80b8c97321b972a4155630f9a75/meson.build#L15)?

Reply:
They definitely [use glib](https://github.com/artemanufrij/regextester/blob/7299891e8f71e113690f7c92dcfb650aed20d52b/src/Widgets/MatchItem.vala#L34) so I would pin them to the second category.